README.md

```diff
@@ -51,19 +81,23 @@ object BehaviorTest {
 
 And results in a compile error, preventing non-runtime-safe code from being executed:
 ```
-test0.scala:7: error: org.random.project.BehaviorTest.Command is used as Akka message but does not extend a trait annotated with org.virtuslab.ash.annotation.SerializabilityTrait.
-Passing an object of class NOT extending SerializabilityTrait as a message may cause Akka to fall back to Java serialization during runtime.
+test0.scala:7: error: org.random.project.BehaviorTest.Command is used as Akka message
+but does not extend a trait annotated with org.virtuslab.ash.annotation.SerializabilityTrait.
+Passing an object of class NOT extending SerializabilityTrait as a message may cause Akka to
+fall back to Java serialization during runtime.
 
 
   def method(msg: Command): Behavior[Command] = ???
                             ^
-test0.scala:6: error: Make sure this type is itself annotated, or extends a type annotated with  @org.virtuslab.ash.annotation.SerializabilityTrait.
+test0.scala:6: error: Make sure this type is itself annotated, or extends a type annotated
+with  @org.virtuslab.ash.annotation.SerializabilityTrait.
   sealed trait Command extends MySerializable
                ^
 ```
 
+
 The compiler plugin only checks the classes in the sbt modules where `AkkaSerializationHelperPlugin` is explicitly enabled.
-It may happen that your base trait (like `MySerializable` in the example) lives in an sbt module like `core` where you do **not** want to enable the plugin
+It may happen that your base trait (like `MySerializable` in the example) may live in an sbt module like `core` where you do **not** want to enable the plugin
 (e.g. for compilation performance reasons).
 You still need to reference `org.virtuslab.ash.SerializabilityTrait` to annotate `MySerializable`, however.
```

Comment:
The usage of `you` and `we` is inconsistent across the text.

I'd prefer using impersonal form wherever natural, or `you` as the second choice; refrain from using `we` as it sounds somewhat... idk, immature (?) in technical writing

Reply:
(esp. that it's unclear to whom `we` refers: _we the authors_? _we = the author + the reader_?)